Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: Content items of a type without an icon get the default icon
+        Given a custom Content Type exists
+          And a Content item of this type exists
+         When I view this Content
+         Then I see the "generic" Content Type icon attached to the title
```

Comment:
I'd remove the "I see": `Then the "generic" Content Type icon is attached to the title` ?